src/test/resources/json/getconsignment_data_all.json

```diff
@@ -4,6 +4,9 @@
       "consignmentid":"b130e097-2edc-4e67-a7e9-5364a09ae9cb",
       "seriesid": "fde450c9-09aa-4ba8-b0df-13f9bac1e587",
       "userid":"4ab14990-ed63-4615-8336-56fbb9960300",
+      "dateTime": "2020-01-01T09:00",
```

Comment:
Is this actually the consignment creation date? Could we give it a name like that? This field will be [renamed in the database](https://national-archives.atlassian.net/browse/TDR-640) at some point, but the API could use a more specific name already.